ircrobots/params.py

```diff
@@ -28,6 +28,12 @@ class ResumePolicy(object):
     address: str
     token:   str
 
+@dataclass
+class ClientTLSCertificate(object):
```

Comment:
I think I'd prefer this to be called a keypair? not sure. thoughts?

Reply:
Yeah I think keypair would actually be the correct name for this.